aws-integrations/ecs-fargate/ecs-fargate-cf.yaml

```diff
@@ -112,12 +112,12 @@ Resources:
       ExecutionRoleArn: !GetAtt ECSExecutionRole.Arn
       NetworkMode: awsvpc
       ContainerDefinitions:
-        - Name: node-app
-          Image: mikebriggs2k/nodeapp:0.0.6
+        - Name: demo
```

Comment:
The reason for using that custom nodeapp instead of nginx was because it is instrumented to submit metrics and traces to the collector. Though not necessary, it does help demonstrate the functionality. Might want to consider a public image that is instrumented, if there is one.

Reply:
The idea behind this CF template is to guide them on how to deploy their ECS tasks and have them submit all three pillars to the platform. Without a container that is instrumented, customers will likely be confused as to why they don't see any metrics or traces in their console. If not my example node-app, then perhaps something else online that we can use that's instrumented.